packages/mobile/src/navigator/types.tsx

```diff
@@ -25,12 +25,10 @@ type NestedNavigatorParams<ParamList> = {
 
 // tslint:disable-next-line: interface-over-type-literal
 export type StackParamList = {
-  [Screens.Account]: undefined
-  [Screens.Analytics]: undefined
   [Screens.BackupComplete]: undefined
   [Screens.BackupIntroduction]:
     | {
-        fromAccountScreen?: boolean
+        fromSettings?: boolean
```

Comment:
i personally prefer using `nextScreen` as opposed to switching to fromSettings

Reply:
The same, as above